static/templates/input_pill.hbs

```diff
@@ -1,4 +1,4 @@
-<div class='pill' data-id='{{ id }}' tabindex=0>
+<div class='pill {{#if not_active}} deactivated-user-pill {{/if}}' {{#if not_active}} title="{{t 'Can\'t send messages to the deactivated user' }}" {{/if}} data-id='{{ id }}' tabindex=0>
```

Comment:
This is a general template for pills everywhere in the app; I think we should make at least the `title` a parameter passed in from the caller.  And probably we should consider having a `deactivated-pill` class rather than `deactivated-user-pill`, since the styling seems like it'd be appropriate for a deactivated stream (or something) as well in the future.

Finally, we should call the variable `deactivated` rather than `not_active`, to avoid adding new terminology.

@m-e-l-u-h-a-n may have some ideas for how to do this nicely.

Reply:
I updated the name. However, I have removed `title` for now. As I think, certain pills having title and certain not will not be a good idea. So, we should pass the title to every pill. Will add it in some other commit.

Reply:
I think we should show title only for deactivated pills, as users may be curious to know why their color is different. From changes in the most recent push, it appears the title is same as the display value for normal pills so that does not add much information here 🤔.

Reply:
Is this only for recipient box pills? Because user pills can also be from
https://github.com/zulip/zulip/blob/87fc2bbe50da15ebcbecfb710e1ef928224bcaa0/static/js/user_pill.js#L60
in `user_pill.js`, which is not changed. 
Most of the pill widgets (Adding members to streams, user group setting, custom profile type with user) use this for creating pills, so if we want this behavior for pills of deactivated users to be the same everywhere we should change this as well to show "(deactivated)" in display value. Maybe we should wait for @timabbott's suggestion here.

Reply:
I have already updated the way. see 4th commit (latest commit) for what change I did. I searched for where these pills and used and added condition based on that.